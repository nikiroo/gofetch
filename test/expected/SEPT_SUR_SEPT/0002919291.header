0Un vagabond soupçonné d%27un double meurtre à la hache en France	null/SEPT_SUR_SEPT/0002919291		70
i  [Monde] 26-09-2018 14:51
i
i    Un  vagabond espagnol, soupçonné d%27un double meurtre perpétré
i    en  juin  dans  le  sud  de la France, a été inculpé et écroué,
i    a-t-on appris mercredi ...
i
