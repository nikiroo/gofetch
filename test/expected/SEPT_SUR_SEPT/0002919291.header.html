<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h1><a href='0002919291.html'>Un vagabond soupçonné d%27un double meurtre à la hache en France</a></h1>
	<div class='details'>([Monde] 26-09-2018 14:51)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Un vagabond espagnol, soupçonné d%27un double meurtre perpétré en juin dans le sud de la France, a été inculpé et écroué, a-t-on appris mercredi ...
	</div>
<hr/>
</div>
</body>
